test/test_steno.py

```diff
@@ -11,33 +11,47 @@
 class StenoTestCase(unittest.TestCase):
     def test_normalize_steno(self):
         cases = (
-        
-        # TODO: More cases
-        ('S', 'S'),
-        ('S-', 'S'),
-        ('-S', '-S'),
-        ('ES', 'ES'),
-        ('-ES', 'ES'),
-        ('TW-EPBL', 'TWEPBL'),
-        ('TWEPBL', 'TWEPBL'),
-        ('19', '1-9'),
-        ('14', '14'),
-        ('146', '14-6'),
-        ('67', '-67'),
-        ('6', '-6'),
-        ('9', '-9'),
-        ('5', '5'),
-        ('0', '0'),
-        ('456', '456'),
-        ('46', '4-6'),
-        ('4*6', '4*6'),
-        ('456', '456'),
-        ('S46', 'S4-6'),
+            # TODO: More cases
+            ('S', 'S'),
+            ('S-', 'S'),
+            ('-S', '-S'),
+            ('ES', 'ES'),
+            ('-ES', 'ES'),
+            ('TW-EPBL', 'TWEPBL'),
+            ('TWEPBL', 'TWEPBL'),
+            ('19', '1-9'),
+            ('14', '14'),
+            ('146', '14-6'),
+            ('67', '-67'),
+            ('6', '-6'),
+            ('9', '-9'),
+            ('5', '5'),
+            ('0', '0'),
+            ('456', '456'),
+            ('46', '4-6'),
+            ('4*6', '4*6'),
+            ('456', '456'),
+            ('S46', 'S4-6'),
+            # Number key.
+            ('#S', '#S'),
```

Comment:
I'd say, if you have numbers, no number symbol. If you don't, you need it.

`#K`: `#K`
`#S`: `1`
`#SK`: `1K`
`STKP#`: `12K3`
`K#`: `#K`

That being said, we could merge this as it's not a regression. If you want to continue down that path, you are welcome to. Let me know :grin: